Centaurus.Domain/WebSockets/Alpha/Info/InfoConnectionManager.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Centaurus.Domain;
+using Centaurus.Models;
+using NLog;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Net.WebSockets;
+using System.Threading.Tasks;
+using System.Linq;
+using Centaurus.Exchange.Analytics;
+using Centaurus.Analytics;
+using System.Threading;
+
+namespace Centaurus.Domain
+{
+    /// <summary>
+    /// Manages all client websocket connections
+    /// </summary>
+    public static class InfoConnectionManager
```

Comment:
The client should be able to sign up for three different data streams:
- recent trades - returns a trade whenever it happens (arguments: `assetId`)
- price history - historical OHLC prices and live updates for a given asset and timeframe (arguments: `assetId` and `timeframe`)
- orderbook - information about current orberbook state with live updates (arguments: `assetId`, `precision`), where precision specifies the rounding precisions for the orderbook entries
- market info - minimalistic market info, including current price, 24h high, 24h low, 24h volume, and 24h price change in percents (arguments: `assetId`)

The client should be able to sign up for any of the above data streams simultaneously in any combination. For example:
- market info for EUR, BTC, and BRENT markets
- orderbook for EUR with 1 digit precision
- OHLC data for BTC (daily) and EUR (hourly)
- trades stream for EUR, CHF, USD

Therefore, all those endpoints should be connected and updated separately, and not in one huge SendMarketUpdates batch.